okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -1056,6 +1068,19 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
     }
   }
 
+  private Future<Void> importModules(ProxyContext pc, String body) {
```

Comment:
The method should be renamed to createModules to parallel the createModule method.
The method should be moved directly below to the createModule method.

Reply:
ok